tox.ini

```diff
@@ -11,6 +11,7 @@ install_command =
 setenv =
    VIRTUAL_ENV={envdir}
    OS_TEST_TIMEOUT=120
+   LANG=en_US.utf8
```

Comment:
We need to set locale for "click" library because without it gives out the error:
Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.

Also this line is mentioned in their project link
https://click.palletsprojects.com/en/7.x/python3/